configurations-conception-communes/signature.md

```diff
@@ -0,0 +1,149 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/signature.html"
+breadcrumbs:
+  - title: "À propos de Canada.ca"
+    link:  "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: "Système de conception de Canada.ca"
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: "Bibliothèque de modèles et de configurations de conception"
+    link: "https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html"
+  - title: "En-tête global"
+    link: "https://conception.canada.ca/configurations-conception-communes/en-tete-general.html"	
```

Comment:
3/3 We could move this file into a folder "connexion" and then apply a default breadcrumb in the _config.yml